### PR TITLE
Maximum of three numbers

### PR DESCRIPTION
**Pull request by MadhuBeesetty**

### CODE REVIEW

File: test/2.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const test = require('../solutions/2.js')
+const testing = (a,b,c) =>{
+  if(test(a,b,c) === a){
```
Comment (songz) [inline test/2.js]: this is a wrong way to test.

File: solutions/2.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const greatest = (a,b,c)=>{
+  if(a>b && a>c){
+    return a
+  }
+  if(b>a && b>c){
+    return b
+  }
+  if(c>a && c>b){
```
Comment (allopez7) [inline solutions/2.js]: song told me just return c here. No if needed

File: solutions/2.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const greatest = (a,b,c)=>{
+  if(a>b && a>c){
+    return a
+  }
+  if(b>a && b>c){
+    return b
+  }else{
```
Comment (songz) [inline solutions/2.js]: no need for else statement.

**Review by allopez7 — APPROVED** [review]

File: test/2.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+const testing = (a,b,c,max) =>{
+  if(test(a,b,c) === max){
+    console.log("this is the maximum");
+  }else{
+    console.log("this is not the maximum");
+  }
+}
+testing(1,1,2,2);
+testing(1,2,1,2);
+testing(2,1,10,2);
```
Comment (songz) [inline test/2.js:11]: 2,1,10,10 - make sure all the test cases are passes.

**Pull request merged**